ticktock-android/lazyzonerules/src/main/java/dev/zacsweers/ticktock/android/lazyzonerules/AndroidLazyZoneRules.java

```diff
@@ -0,0 +1,19 @@
+package dev.zacsweers.ticktock.android.lazyzonerules;
+
+import android.content.Context;
+import dev.zacsweers.ticktock.android.AssetsZoneDataLoader;
+import dev.zacsweers.ticktock.runtime.TickTockPlugins;
+
+public class AndroidLazyZoneRules {
```

Comment:
Should this class maybe also offer caching mechanism? That way you consumers only need a single class. Another question is if the startup artifact should do any caching.

Reply:
Good points. Let's look in a followup https://github.com/ZacSweers/ticktock/issues/8